backend/song/tests/test_views.py

```diff
@@ -0,0 +1,277 @@
+import pytest
+
+from django.urls import reverse
+
+from rest_framework.status import (
+    HTTP_200_OK,
+    HTTP_201_CREATED,
+    HTTP_204_NO_CONTENT,
+    HTTP_403_FORBIDDEN,
+)
+from rest_framework.test import APIClient
+
+from backend.song.models import Artist
+
+
+pytestmark = pytest.mark.django_db
+
+
+class TestCategory:
+
+    def generic_get_request(self, url):
+        client = APIClient()
+        return client.get(url)
+
+    def test_create_category(self, auth, create_category):
+
+        assert create_category.status_code == HTTP_201_CREATED
+        assert create_category != {}
```

Comment:
which one is pythonic between testing using` {}` or using truly and falsy concepts?

Reply:
@tadomikikuto-bit